<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>successlinks</class>
 <widget class="QDialog" name="successlinks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Success</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/logo1.png</normaloff>:/icon/logo1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(249, 247, 241);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>80</y>
     <width>56</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Baskerville Old Face&quot;;
color: rgb(58, 80, 116);</string>
   </property>
   <property name="text">
    <string>Link:</string>
   </property>
  </widget>
  <widget class="QLabel" name="all">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>30</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Baskerville Old Face&quot;;
color: rgb(58, 80, 116);</string>
   </property>
   <property name="text">
    <string>All links are active now.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>150</y>
     <width>105</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0 rgb(95, 167, 175), stop: 1 rgb(58, 80, 116));
color: rgb(249, 247, 241); 
font: 14pt &quot;Baskerville Old Face&quot;;
border-radius:15px;</string>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="link">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>80</y>
     <width>370</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Baskerville Old Face&quot;;
color: rgb(58, 80, 116);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="share1">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>30</y>
     <width>356</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Baskerville Old Face&quot;;
color: rgb(58, 80, 116);</string>
   </property>
   <property name="text">
    <string>All links are active now for a limit of another</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="share2">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>30</y>
     <width>40</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Baskerville Old Face&quot;;
color: rgb(58, 80, 116);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="share3">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>30</y>
     <width>56</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Baskerville Old Face&quot;;
color: rgb(58, 80, 116);</string>
   </property>
   <property name="text">
    <string>shares.</string>
   </property>
  </widget>
  <widget class="QLabel" name="share4">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>60</y>
     <width>356</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Baskerville Old Face&quot;;
color: rgb(58, 80, 116);</string>
   </property>
   <property name="text">
    <string>All links have been disabled</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="timer1">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>30</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Baskerville Old Face&quot;;
color: rgb(58, 80, 116);</string>
   </property>
   <property name="text">
    <string>All links are active now until:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="timer2">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>30</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Baskerville Old Face&quot;;
color: rgb(58, 80, 116);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
